.github/workflows/reusable-e2e.yml

```diff
@@ -322,6 +322,7 @@ jobs:
           packages: |
             packages/react-native
             packages/react-core
+            packages/react-core-notifications
```

Comment:
Will this cause any issues if the package doesn't exist?

Reply:
The next PR this workflow runs on is the PR where we add all the new packages including `react-core-notifications`.

But in general, looking at the check-changed-packages code, when running grep on the changed files, there just won't be any files within `packages/react-core-notifications` for now.